web/src/components/tasks/business-formation/name/BusinessNameStep.tsx

```diff
@@ -127,7 +127,7 @@ export const BusinessNameStep = (): ReactElement => {
           </div>
         </WithErrorBar>
       </form>
-      {error != null && (
+      {error && (
```

Comment:
<img width="410" alt="Screen Shot 2023-03-21 at 11 02 31 PM" src="https://user-images.githubusercontent.com/22647707/226791198-a5efddb0-e119-4d6a-a8a2-3e22f86804f8.png">

`SearchBusinessNameError` is a string. Dropped this comparison because the strict comparison causes the build to fail and strings are truthy in JS.